src/main/java/org/jenkinsci/plugins/workflow/steps/EnvStep.java

```diff
@@ -150,11 +153,37 @@
                     }
                 }
                 return b.toString();
+            } else if (overrides instanceof Map) {
+                return ((Map<?, ?>) overrides).keySet()
+                    .stream()
+                    .filter(e -> e instanceof String)
+                    .map(Object::toString)
+                    .collect(Collectors.joining(", "));
```

Comment:
Not covered by tests AFAICT?

Reply:
Definitely covered by tests as it was failing before I created the converter 😆

Reply:
Meaning tests verify that this method returns normally when a map is used, not that the return value is correct.

Reply:
No there is a test that looks at the output.
https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/105/files#diff-c2e520244ad09761e2187513ef5c751cR111

Reply:
To the two tests I added now I am testing the output of the function.